### PR TITLE
Add stats endpoint

### PR DESCRIPTION
**Pull request by szlend**

### CODE REVIEW

File: source/endpoints/stats.md.erb [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Data stats
+
+Calculates statitics for a specified URL or View in a given date range.
```
Comment (jurgenwerk) [inline source/endpoints/stats.md.erb]: Typo

File: source/endpoints/stats.md.erb [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+## List keywords
```
Comment (jurgenwerk) [inline source/endpoints/stats.md.erb]: List keywords?

File: source/endpoints/stats.md.erb [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+`GET https://api.nightwatch.io/api/v1/keyword_stats?access_token=ACCESS_TOKEN`
+
+### URL Parameters
+
+Parameter       | Description
+--------------- | -----------
+start_date      | The starting date in YYYY-MM-DD format (e.g. 2019-01-01)
+end_date        | The ending date in YYYY-MM-DD format (e.g. 2019-01-31)
+url_id          | The URL id to calculate stats for
+dynamic_view_id | The dynamic view id to calculate stats for (optional)
```
Comment (jurgenwerk) [inline source/endpoints/stats.md.erb]: I would mention here it's mandatory to either provide a url id or a view id.

**Review by jurgenwerk — CHANGES_REQUESTED** [review]

**Comment by szlend** [comment]
@matixmatix Fixed

**Review by jurgenwerk — APPROVED** [review]

**Pull request merged**